packages/common/src/ui/forms/Modal/ModalInput.tsx

```diff
@@ -35,6 +37,7 @@ export const ModalInput: React.FC<ModalInputProps> = ({
     >
       <BasicTextInput
         disabled={disabled}
+        value={value}
```

Comment:
since we're not unmounting things when we ok and next, the default value doesn't cooperate